dashboard/app/lib/javascripts/dashboard/dispatcher.js

```diff
@@ -18,6 +18,51 @@ var Dispatcher = extend({
 		this.dispatch(extend({
 			source: "APP_EVENT"
 		}, event));
+	},
+
+	handleServerEvent: function (event) {
+		var e = extend({}, event);
+		switch (event.object_type) {
+			case 'app':
+				e.name = 'APP';
+			break;
+
+			case 'app_release':
+				e.name = 'APP_RELEASE';
+			break;
+
+			case 'app_deletion':
+				e.name = 'APP_DELETED';
+			break;
+
+			case 'route':
+				e.name = 'ROUTE';
+			break;
+
+			case 'route_deletion':
+				e.name = 'ROUTE_DELETED';
```

Comment:
It looks like this can be simplified to just upcase, s/deletion/deleted, and ignore unknown events?